meshroom/core/node.py

```diff
@@ -1085,19 +1110,24 @@ def __init__(self, nodeType, position=None, parent=None, **kwargs):
         for attrDesc in self.nodeDesc.outputs:
             self._attributes.add(attributeFactory(attrDesc, None, True, self))
 
+        for attrDesc in self.nodeDesc.internalInputs:
+            self._internalAttributes.add(attributeFactory(attrDesc, None, False, self))
+
         # List attributes per uid
         for attr in self._attributes:
             for uidIndex in attr.attributeDesc.uid:
                 self.attributesPerUid[uidIndex].add(attr)
 
         self.setAttributeValues(kwargs)
+        self.setInternalAttributeValues(kwargs)
 
     def setAttributeValues(self, values):
         # initialize attribute values
         for k, v in values.items():
-            attr = self.attribute(k)
-            if attr.isInput:
-                attr.value = v
+            if self.hasAttribute(k):
```

Comment:
I'm not sure if it's correct for setAttributeValues on a non-existing attribute to avoid the raise of an exception.

Reply:
Fixed.